redis/Cargo.toml

```diff
@@ -74,11 +82,16 @@ geospatial = []
 json = ["serde", "serde_json"]
 cluster = ["crc16", "rand"]
 script = ["sha1_smol"]
-tls = ["native-tls"]
+tls-native-tls = ["native-tls"]
+tls-rustls = ["rustls", "webpki-roots"]
```

Comment:
I think we should use `rustls-native-certs` as the default, and make `webpki-roots` the configurable option.

Reply:
Just did it 👍🏽.